package.json

```diff
@@ -6,7 +6,7 @@
     "build": "turbo run build",
     "dev": "turbo run dev --no-cache --continue",
     "lint": "turbo run lint",
-    "clean": "turbo run clean && rm -rf node_modules"
+    "clean": "turbo run clean && rm -rf node_modules && rm -rf .turbo"
```

Comment:
Ran into an issue with rc v2 where turbo cache skipped the build after clearing build artifacts, so this fixes that. Probably gonna nuke turborepo in a follow, no reason to even need it here tbh.